MCSync/frmConfig.cs

```diff
@@ -121,7 +107,16 @@ private void cmdResourcePacksWhitelist_Click(object sender, EventArgs e)
             whitelist.Text = "Resource Packs Whitelist";
             whitelist.WhitelistName = "resourcepacks_whitelist";
             whitelist.lblIntro.Text = "Edit your Resource Packs whitelist.";
-            whitelist.ShowDialog();
+
+            whitelist.TopLevel = false;
+            whitelist.ControlBox = false;
+            whitelist.Dock = DockStyle.Fill;
+            whitelist.FormBorderStyle = FormBorderStyle.None;
```

Comment:
Should we actually just set these on the form itself rather than doing it here?

Reply:
Fixed